web/src/components/CityEntry.tsx

```diff
@@ -31,50 +31,44 @@ type CityEntryProps = {
   language: string
   city: CityModel
   filterText: string
-  theme: DefaultTheme
 }
 
-class CityEntry extends React.PureComponent<CityEntryProps> {
-  getMatchedAliases = (city: CityModel, normalizedFilter: string): Array<string> => {
-    if (city.aliases && normalizedFilter.length >= 1) {
-      return Object.keys(city.aliases).filter(alias => normalizeString(alias).includes(normalizedFilter))
-    }
-    return []
-  }
+const CityEntry = ({ filterText, city, language }: CityEntryProps): ReactElement => {
```

Comment:
Only converted from class to function component and use `useTheme` hook.

Reply:
We have the `<ThemeProvider>` and the `useTheme` hook. As our WHoe app is wrapped in the <ThemeProvider>, why do we still need/use the hook?

Reply:
Sorry but I don't understand the question :see_no_evil: Can you try again?

If you are asking why we need to use the hook/need the theme in this case, see line 63. We have to pass the theme color there.